misc/chip_data.json

```diff
@@ -1864,7 +1864,7 @@
     },
     "15e42063-cbae-40f9-a9e4-8226e83217ec": {
       "ReadonlyName": "Emitter Set Color",
-      "Description": "",
+      "Description": "Sets a color for the emitter",
```

Comment:
maybe this should be something like "Sets a color for the particles emitted"? I like your size description because it references the particles themselves that are getting changed.